packages/fxa-settings/README.md

```diff
@@ -1,75 +1,186 @@
+
+For example:
+
+```tsx
+// my-component.tsx
+<button className="block w-full py-2 text-center mt-6 bg-grey-10 border border-grey-200 transition duration-150 rounded hover:border-grey-200 hover:bg-grey-100 hover:text-grey-400 hover:transition hover:duration-150 active:border-grey-400 active:bg-grey-100 active:text-grey-400 active:transition active:duration-150">
+  Click here!
+</button>
+
+// my-other-component.tsx
+<a href="#" className="block w-full py-2 text-center mt-6 bg-grey-10 border border-grey-200 transition duration-150 rounded hover:border-grey-200 hover:bg-grey-100 hover:text-grey-400 hover:transition hover:duration-150 active:border-grey-400 active:bg-grey-100 active:text-grey-400 active:transition active:duration-150">
+  Click here!
+</a>
+```
+
+This makes more sense as a new utility class instead of a component because sometimes we style a button and other times a link. Let's extract this instead into `ctas.scss` with a fitting name:
+
+```scss
+// ctas.scss
+.cta-neutral {
+  @apply block w-full py-2 text-center mt-6 bg-grey-10 border border-grey-200 transition duration-150 rounded;
+
+  &:hover {
+    @apply border-grey-200 bg-grey-100 text-grey-400 transition duration-150;
+  }
+
+  &:active {
+    @apply border-grey-400 bg-grey-100 text-grey-400 transition duration-150;
+  }
+}
+```
+
+Assuming this package uses `postcss-import`, all we need to do now is `@import` ctas.scss into our `tailwind.css` file and apply our new utility class:
+
+```tsx
+// my-component.tsx
+<button className="cta-neutral">
+  Click here!
+</button>
+
+// my-other-component.tsx
+<a href="#" className="cta-neutral">
+  Click here!
+</a>
+```
+
+Note that the above doesn't have to be in an external SCSS file, `@apply` and other PostCSS at-rules can be used in `.css` files, but SCSS still comes in handy for nesting.
+
+Keep in mind that custom component classes are not scoped per component and some line of thought should go into custom class names to avoid naming collisions. Generally, you should match the class name to the name of your component and it's recommended to use `[component]-[descriptor]` style (e.g. using a shared UnitRow to display secondary emails could be `unit-row-secondary-emails`). Also, use `@apply` exclusively if possible, keep the selector one level deep, never use `!important` to avoid specificity collisions, and don't use `&-` overzealously as it makes classes difficult to search for (one level deep is fine). These recommendations generally also apply to custom styles.
+
+See the [Tailwind docs on this subject](https://tailwindcss.com/docs/extracting-components/) for more detailed information and examples.
 
-Here are a few additional things to keep in mind when developing with Tailwind:
+#### PurgeCSS
```

Comment:
Is this intended to be a separate section? PurgeCSS [+PostCSS] is effectively part of our Tailwind setup

Reply:
Hmm well, it still falls under the `### Styling components` header. It feels fitting there, I do also mention this higher up the doc:
> Note that if you run a build command to test a production build, you'll need to make an update to one of these files with `pm2` running or manually run `yarn build-postcss` to rebuild the dev version containing all available classes (see the "PurgeCSS" section for more details).